### PR TITLE
update install doc: helm get package version from Erda VERSION

### PR DESCRIPTION
**Pull request by sfwn**
#### What type of this PR

/kind document

#### What this PR does / why we need it:

Helm get package version from Erda VERSION, not fixed 0.1.0.

#### Need cherry-pick to release versions?

/cherry-pick master

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by sfwn** [comment]
https://github.com/erda-project/erda-release/pull/8

File: docs/guides/deploy/How-to-install-Erda-zh.md [hunk 1]
```diff
@@ -113,13 +113,13 @@
...
+   helm install package/erda-base-$(cat VERSION).tgz --generate-name
    
    # 安装 erda-addons
-   helm install package/erda-addons-0.1.0.tgz --generate-name
+   helm install package/erda-addons-$(cat VERSION).tgz --generate-name
    
    # 安装 erda
-   helm install package/erda-0.1.0.tgz --generate-name
+   helm install package/erda-$(cat VERSION).tgz --generate-name
    ```
```
Comment (Effet) [inline docs/guides/deploy/How-to-install-Erda-zh.md:123]: have you mentioned where this script executed from?

**Review by Effet — CHANGES_REQUESTED** [review]

**Comment by Effet** [comment]
and why this pr targeting to release/1.0 not master

**Comment by sfwn** [comment]
> and why this pr targeting to release/1.0 not master

Make sure release/1.0 is ok firstly.

**Comment by Muzry** [comment]
lgtm

File: docs/guides/deploy/How-to-install-Erda-zh.md (hunk 1 above)
Comment (sfwn) [reply docs/guides/deploy/How-to-install-Erda-zh.md:123]: Doc above already comment out. And `VERSION` file is in the same folder as `package` dir. So no new broken things.

**Pull request merged**